example/stpaul/project_cards/6_test_project_card.yml

```diff
@@ -0,0 +1,21 @@
+name: 6_test_project_card
```

Comment:
I don't think we need to add this here. why not change the value of "project" to "name"? We can save the path where the project card was read from as the filename...

Reply:
yeah, that works too!

Reply:
I'll update the code to reflect this change.